### ISSUE TITLE
[Request] Shared links

### ISSUE THREAD

**Issue by MickeMM** [paulijar/music#7]
I have a shared folder with music and videos. Is it possible to add music app to shared folders?
I have search but couldent find anything. The gallery app i use can be used for non-logged in and it would be perfect if music app would do the same. Thanks! :)

**Comment by paulijar** [comment]
The Music app is supposed to work in the shared folders without any user actions. I haven't tested it recently, though. Which version of ownCloud or Nextcloud are you using?

**Comment by MickeMM** [comment]
Here is one of my share. 
[Share](https://backup.vimla-tipslank.se/index.php/s/N08zJEOM6yK31Rr)

I use the latest owncloud. 

![Screenshot (1)](https://user-images.githubusercontent.com/59674728/181376359-1713997a-01a7-447c-bc20-46db1cd88146.png)

**Comment by MickeMM** [comment]
Sorry! My misstake. 
It works but i belive that i should look like when im logged in. With album selection and all that :)

I press one song and the player started. :)

[Test](https://backup.vimla-tipslank.se/index.php/s/5rg7nIPY4VyHptW)

**Comment by paulijar** [comment]
Yes, we only support the "lite" player on shared folders, so the Music works there the same way as within the Files app of the logged in user.

Frankly, it has never appeared to me that it could make sense to have the full-blown Music app on the shared folder. Or that this kind of thing could be technically possible. But as that works with the Gallery, yes, it might be within the realm of possibility. However, I presume it would require quite much effort to make it happen.

**Comment by Soko1201** [comment]
Hello !!! 

When I share a song and the link is clicked, the cover image no longer shows. 
Otherwise the picture was always there. Will it work again?

Thanks

My NextColud 26.0.1 Version
![700-1024](https://user-images.githubusercontent.com/110041577/233602799-a147bac8-17f7-4fce-b10c-baad9bec12ea.jpg)
